index.html

```diff
@@ -92,6 +92,26 @@ <h4 class="subheading">
     </div>
   </div>
 
+  <div class="contribute">
+    <div class="container-sml">
+      <div class="flex text-center center">
+        <div class="col-10 center">
+          <div class="gulp-image">
+            <img class="logo-img-lg" src="img/gulp-white-logo.svg">
+          </div>
+          <div class="contribute-copy">
+            <h1 class="text-uppercase contribute-item">Become a part of it!</h1>
+            <p class="contribute-item">
+              With your support you'll help us cover all the expenses our project needs to keep going!
+            </p>
+            <a class="ctas-button contribute-item"
+              href="https://opencollective.com/gulpjs">Become a backer or sponsor</a>
```

Comment:
"Become a backer"

Not really useful to delineate since the page they'll go to has more info.